util.go

```diff
@@ -133,7 +134,7 @@ func list(fs *Filestore, verify bool, key mh.Multihash) *ListRes {
 
 func listAll(fs *Filestore, verify bool) (func() *ListRes, error) {
 	q := dsq.Query{}
-	qr, err := fs.fm.ds.Query(q)
+	qr, err := fs.fm.ds.Query(context.Background(), q)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
Either the lambda returned below or ListAll() should take a context. Probably ok to leave it like this for now.